frontend/app/models/person.js

```diff
@@ -6,7 +6,7 @@ const Person = DS.Model.extend({
   birthdate: DS.attr("date"),
   picturePath: DS.attr("string"),
   location: DS.attr("string"),
-  maritalStatus: DS.attr("string"),
+  maritalStatus: DS.attr("string", { defaultValue: "single" }),
```

Comment:
is this really needed?, since it wasn't needed before, or was this a bug?

Reply:
This is necessary now

Reply:
Maybe elaborate a bit don't just say yes? :grin:

Reply:
It's easier and more readable to set the maritalStaus default value right in the model itself.
Is was defined in the logic of the selector previously.